src/main/java/nl/tudelft/cse1110/andy/execution/ExecutionFlow.java

```diff
@@ -15,9 +15,14 @@
     private LinkedList<ExecutionStep> steps;
     private final ResultWriter writer;
 
-    private ExecutionFlow(Context ctx, ResultBuilder result, ResultWriter writer) {
+    private ExecutionFlow(Context ctx, ResultBuilder result, ResultWriter writer, boolean addSecurityManager) {
```

Comment:
Can we have two methods here, buildWithSecurityChecks and build ?

Internally, we can use a Boolean to differentiate and avoid duplicate code, but for the clients of the API, i would avoid Boolean flags like this one.

Reply:
If that's too much work given the MetaStep, then it's ok for now.

Reply:
Updated this to have two methods, `build` and `buildWithoutSecurityChecks`. I do not understand what is extra about the `MetaStep` but that might just be my misunderstanding of how the meta step works